### ISSUE TITLE
Contact form does not work due to recaptcha 

### ISSUE THREAD

**Issue by tmaier** [BlackrockDigital/startbootstrap-clean-blog-jekyll#167]
I wondered why my contact form does not work.

I checked the network requests. 
<img width="1059" alt="bildschirmfoto 2018-11-10 um 15 28 33" src="https://user-images.githubusercontent.com/350038/48302447-58a70600-e4fd-11e8-9c61-1a80dddcdc4a.png">

The request itself looked quite ok
<img width="917" alt="image" src="https://user-images.githubusercontent.com/350038/48302455-7c6a4c00-e4fd-11e8-9b98-78dcdd7b3c99.png">

But the HTML returned by formspree does not look that nice.
<img width="694" alt="image" src="https://user-images.githubusercontent.com/350038/48302465-a02d9200-e4fd-11e8-8668-991b4c8aeb0b.png">

As you can see, it requests for a captcha to pass.

Please provide a way how to handle this within the theme.

**Comment by tmaier** [comment]
A poor mans version could be to get rid of the ajax at all and to use features, like `_next` provided by formspree to show a success message

* `_next`: https://formspree.io/docs/

**Comment by reidhock** [comment]
I'd love to know if you guys have fixed this issue....

**Comment by tmaier** [comment]
@reidhock, you could replace all the ajax magic with a simple HTML-only form using formspree

**Comment by reidhock** [comment]
@tmaier I'm very new to github... I don't know what "creating a PR" means haha. I will def try what you said tho.

**Comment by yasthil** [comment]
I've done the same as @reidhock.

Formspree does not allow form submissions using ajax from free accounts. 
After changing the ajax call to include a dataType attribute:
`$.ajax({
	url: "//formspree.io/{{ site.email }}",
	type: "POST",
	dataType: "json",
	data: {
	  name: name,
	  phone: phone,
	  email: email,
	  message: message
	}
`
I received the following response:
`{"error":"To prevent spam, free forms cannot submit via AJAX."}`

**Comment by Cramenorn** [comment]
@yasthil exactly, you need formspree's premium account to make Http requests work correctly. In my case the request returned 401.

**Comment by davidtmiller** [comment]
It looks like the formspree changed their free tier to no longer allow this to work. Can anyone confirm?

**Comment by Cramenorn** [comment]
@davidtmiller I can confirm that Formspree changed the free tier, you need to use the gold or platinum subscription to use HTTP requests:

![image](https://user-images.githubusercontent.com/43040746/85848471-78aa3c80-b7a9-11ea-82d4-21dda2b7f4a9.png)


![image](https://user-images.githubusercontent.com/43040746/85848534-98d9fb80-b7a9-11ea-93b3-a08fde08eb6d.png)

**Comment by ciraben** [comment]
For anyone interested, I adjusted the contact form to work with Formspree free tier [here](https://github.com/ciraben/startbootstrap-clean-blog-jekyll/tree/contact-fix).

**Comment by max-torch** [comment]
> For anyone interested, I adjusted the contact form to work with Formspree free tier [here](https://github.com/ciraben/startbootstrap-clean-blog-jekyll/tree/contact-fix).

This took me a step in the right direction. What I did was:
1. Copy ciraben's contact.html
2. Register a Formspree account
3. Verify my email with Formspree
4. Make a new form in Formspree
5. Copy the URL of the new form
6. In the contact form's HTML tag, replace the URL in `action="https://formspree.io/{{ site.email }}` with the copied URL
7. Add name="message" to the textarea tag attribute of the message field.

It works for me now.
![image](https://user-images.githubusercontent.com/41433185/119257002-f236ee80-bbf5-11eb-9639-e31a11413040.png)
Thank you @ciraben

**Comment by maroskukan** [comment]
Thanks @ciraben and @max-torch. As you described few things need to be adjustment.  My approach was:

Add the following lines to `_config.yml`

```yml
formspree:
  form_id: <your-formspree-form-id>
```
 
Adjust the form action url in `contact.html` to
```html
action="https://formspree.io/f/{{ site.formspree.form_id }}"
```

And in same file add textarea name which was missing, therefore not processed by formspree.

```html
<textarea name="message" rows="5" class="form-control" placeholder="Message" id="message" required
                data-validation-required-message="Please enter a message."></textarea>
```
